packages/client/src/intros/shared/Intro.tsx

```diff
@@ -76,25 +76,24 @@ export default ({
       {exceptions.length > 0 && (
         <>
           <Heading styleAs="h4" forwardedAs="h3">
-            {t("introPage.common.exceptions title")}
+            <em>{t("introPage.common.exceptions title")}</em>
```

Comment:
The font should be a bit smaller and this title should have more weight. 

![image](https://user-images.githubusercontent.com/7781865/110338089-e7bd7d80-8026-11eb-8842-d4e9b8ed6a7c.png)